draft-ietf-taps-interface.md

```diff
@@ -3305,7 +3316,7 @@ this is offered by the `Close` Action with slightly changed semantics in line wi
 * "Abort without delivering remaining data, causing an event informing the application on the other side" and "Abort without delivering remaining data, not causing an event informing the application on the other side":
 this is offered by the `Abort` action without promising that this is signaled to the other side. If it is, a `ConnectionError` Event will fire at the peer ({{termination}}).
 
-* "Reliably transfer data, with congestion control", "Reliably transfer a message, with congestion control" and "Unreliably transfer a message":
+* "Reliably transfer data, with congestion control", "Reliably transfer a message, with congestion control" and "Transfer a message without reliability guarantees":
```

Comment:
```suggestion
* "Reliably transfer data, with congestion control", "Reliably transfer a message, with congestion control" and "Unreliably transfer a message":
```

Like it or not, but this is a reference to a specific term from RFC 8923, and should therefore be kept as it is. The point of this appendix is to enable back-tracking to the original spec, via 8923 => 8303 or 8304.